### PR TITLE
Add double down enhancment

### PR DESCRIPTION
**Pull request by UshalNaidoo**
Added the double down enhancement, done in a separate branch than the split functionality

### CODE REVIEW

File: src/com/swap/blackjack/MainActivity.java [hunk 1]
```diff
@@ -107,982 +107,1008 @@ public void onCreate(Bundle savedInstanceState) {
...
+
+  private void btnDoubleDownClick() {
+    Thread t = new Thread() {
+      public void run() {
+        runOnUiThread(new Runnable() {
+          public void run() {
+            try {
+              sleep(700);
+              _money = _money - _bet;
+              _bet = _bet * 2;
```
Comment (swapagarwal) [inline src/com/swap/blackjack/MainActivity.java]: Update the money and bet values in the layout as well.

**Comment by swapagarwal** [comment]
:+1: Nice, double down is working as expected. Just take a look at the line note and it should be good to merge!

File: src/com/swap/blackjack/MainActivity.java (hunk 1 above)
Comment (Randyfreak) [inline src/com/swap/blackjack/MainActivity.java]: Done

**Comment by swapagarwal** [comment]
This is ready to be merged! (waiting for #3 to finish to handle any compatibility issues)

**Comment by swapagarwal** [comment]
Hey, can you pull the master branch and apply these changes on top of that (to resolve any conflicts)?

**Pull request closed without merging**